com.github.vikdevelop.pythonpackager.yaml

```diff
@@ -0,0 +1,25 @@
+app-id: com.github.vikdevelop.pythonpackager
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: pythonpackager.sh
+finish-args:
+  - --share=ipc
+  - --filesystem=home
```

Comment:
it doesn't use any UI?

Reply:
No, this is a console application. In .desktop file is enabled option `terminal=true`.